packages/types/src/extension-types.ts

```diff
@@ -34,6 +36,7 @@ export interface IExtension<TCreationParameters = any> {
     actionSigner: Identity.IIdentity,
     timestamp: number,
   ) => RequestLogic.IExtensionStates;
+  createCreationAction: (parameters: TCreationParameters) => IAction<TCreationParameters>;
```

Comment:
This method is public on all PN, but it wasn't exposed in this interface.
Exposing it allows to call it [here](https://github.com/RequestNetwork/requestNetwork/pull/1405/files#diff-37ce446eeafd558b24a48b848941f960cfcf8703c119d6ffc3767318568d12e0R45)